.github/workflows/all-modules-add-and-buildable.yaml

```diff
@@ -8,6 +8,12 @@ on:
   schedule:
     - cron: '15 3 * * *'
 
+  # Triggers the workflow on push or pull request events but only for the master branch
```

Comment:
@olehermanse wasn't sure why this wasn't enabled on branches and pull_requests before. Seems like a pretty good idea.

Reply:
@craigcomstock IIRC we have hit API rate limits in the past, and restricted how often to do some of these things.

Reply:
Ok. So leave as-is as a nightly.